src/Route.jsx

```diff
@@ -0,0 +1,24 @@
+import { BrowserRouter, Route as R, Routes } from "react-router-dom";
+import Home from "./pages/Home/Home";
+import Login from "./pages/Login/Login";
+import Card from "./components/Card/Card";
+import Main from "./pages/Main/Main";
+import Folder from "./pages/Folder/Folder";
+
+const Route = () => {
```

Comment:
제가 이전 주 멘토링 때 빠르게 라우터가 무엇인지 알려드리고 싶어서 구현했던 곳에서 좋지 않은 방식이 있었습니다.

Route를 Router로 수정해주시고 `react-router-dom`에서 import  해온 Route의 이름을 변경하지 말고 사용해주세요 😢 
```suggestion
import { BrowserRouter, Routes, Route } from "react-router-dom";

const Router = () => {
```

Reply:
감사합니당~!